### DISCUSSION TITLE
Logging activities in the Event Log

### DISCUSSION THREAD

**Discussion by gurosatherHeco** [Volue-Public/energy-mesh-python#336] [Q&A] [answered]
Hi. In our WO it is described that activities not authorized to the user should be logged to the Event Log. It seems like such activities are not logged to the Event Log? For instance, for an user not authorized to read time series activities where the user tries to read time series are not logged.
![image](https://user-images.githubusercontent.com/82815172/208401997-f156566d-65c8-4e6f-a184-9d1534d08c6e.png)
@alftester

**Comment by alftester** [comment]
We tried writing to a timeseries with a user that only had read permissions, but did not see any trace of this in the Powel Event Log.

**Reply by tnoczyns-volue** [reply]
I created issue for this: https://github.com/Volue-Public/energy-mesh-python/issues/338. I'll provide more information after planning session on January 9th.

**Reply by alftester** [reply]
Thank you, We think "identification of who/writes/what" is more important than this issue.

**Comment by tnoczyns-volue** [comment, marked as answer]
Feature implemented in Mesh server version 2.7.
